eq_cir_proxy_service/main.py

```diff
@@ -12,6 +13,9 @@
 )
 from eq_cir_proxy_service.routers import instrument_router
 
+# Load .env file
+load_dotenv()
```

Comment:
Change it to: `load_dotenv(".env")` for safety?
```suggestion
load_dotenv(".env")
```